### ISSUE TITLE
Release v1.32.4 does not include `deno-linux-arm64.zip`

### ISSUE THREAD

**Issue by liangchunn** [LukeChannings/docker-deno#17]
It seems like the release `v1.32.4` does not have the pre-built deno binary: https://github.com/LukeChannings/deno-arm64/releases/tag/v1.32.4

This was causing an issue on my Docker image that results in:
```
 > [14/22] RUN curl -s https://gist.githubusercontent.com/LukeChannings/09d53f5c364391042186518c8598b85e/raw/ac8cd8c675b985edd4b3e16df63ffef14d1f0e24/deno_install.sh | sh:
curl: (22) The requested URL returned error: 404  
```

**Comment by LukeChannings** [comment]
Yep, there’s a blocker to compiling for Linux ARM. See [this issue](https://github.com/littledivy/fastwebsockets/issues/2)

I will wait for a fixed release of the faster sockets crate and then patch the release.

**Comment by LukeChannings** [comment]
I've patched v1.32.4 to use `fastwebsockets@0.1.3`, which fixes this issue.

**Comment by liangchunn** [comment]
@LukeChannings Thank you so much! 🙏
